README.md

```diff
@@ -119,8 +123,9 @@ use this example [rpi-xcompile.Dockerfile](https://github.com/twistedfall/opencv
 docker build -t rpi-xcompile -f tools/docker/rpi-xcompile.Dockerfile tools
 ```
 
-Building this image requries `qemu-arm` to be present on the host system and the corresponding `binfmt-misc` set up (see
-e.g. https://wiki.debian.org/QemuUserEmulation, only `Installing packages` should be enough).
+Building this image requries `qemu-arm` to be present on the host system and the corresponding `binfmt-misc` set up 
+- see e.g. https://wiki.debian.org/QemuUserEmulation, only `Installing packages` should be enough for debian-based distros,
+- for opensuse, install `qemu-linux-user` via zypper to set up the host correctly.
```

Comment:
Can you please specify what is meant by "set up the host correctly"?

Reply:
fair point. 

```suggestion
- for opensuse, run [`zypper install qemu-linux-user`](https://software.opensuse.org/package/qemu-linux-user) to run user space binaries and libraries meant for another architectures.
```

Is this better? This is the wording from the package that needs to be installed. opensuse does not provide `binfmt-support`...

Reply:
Hmm, I've misread the initial message, I've read "*and* set up the host correctly" thus my confusion, sorry about that! It's good as it is, thank you!